scripts/build-win.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const tp = require('teen_process');
+const log = require('fancy-log');
+
+const APKS_DIR = './apks';
+const GRADLE_DIR = require('path').resolve('./');
+const APK_FILENAME = '/ApiDemos-debug.apk';
+
+async function gradleExecution() {
```

Comment:
function name should contain a verb

Reply:
You mean 'buildAPK' in example?

Reply:
It is a general convention. I mean `gradleExecution` should be changed to `runGrandleBuild` or something like that